adapters/IronSource/IronSourceAdapter/GADMAdapterIronSource.m

```diff
@@ -13,194 +13,14 @@
 // limitations under the License.
 
 #import "GADMAdapterIronSource.h"
-#import "GADMAdapterIronSourceConstants.h"
-#import "GADMAdapterIronSourceInterstitialDelegate.h"
-#import "GADMAdapterIronSourceUtils.h"
-#import "ISMediationManager.h"
-
-@interface GADMAdapterIronSource () <GADMAdapterIronSourceInterstitialDelegate> {
-  // Connector from Google Mobile Ads SDK to receive interstitial ad configurations.
-  __weak id<GADMAdNetworkConnector> _interstitialConnector;
-}
-
-/// Yes if we want to show IronSource adapter logs.
-@property(nonatomic, assign) BOOL isLogEnabled;
-
-/// Holds the ID of the ad instance to be presented.
-@property(nonatomic, copy) NSString *instanceID;
-
-/// Holds the state of the ad instance to be presented.
-@property(nonatomic, copy) NSString *instanceState;
-
-@end
+#import "GADMediationAdapterIronSource.h"
 
 @implementation GADMAdapterIronSource
+/// TODO(Google): Remove this class once Google's server points to GADMediationAdapterIronSource
+/// directly to ask for a Interstitial.
```

Comment:
Nit, can remove this. All the `load<format>ForAdConfiguration` method should be in the `GADMediationAdapterIronSource` class.